.github/workflows/pr.yml

```diff
@@ -116,10 +195,27 @@ jobs:
         run: pip install sqlfluff
       - name: Lint template
         run: make sql_lint_template
+  check-versions-match:
+    name: Check versions match
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # ratchet:actions/checkout@v3
+      - name: Install jq
+        run: sudo apt-get install jq
+      - run: make check_versions_match
   prettier-yaml-lint:
     name: Prettier YAML Lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # ratchet:actions/checkout@v4
-      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # ratchet:actions/setup-node@v4
+      - name: Setup Node
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # ratchet:actions/setup-node@v3
+      - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # ratchet:pnpm/action-setup@v2
+        name: Install pnpm
+        id: pnpm-install
+        with:
+          version: 8
+          run_install: false
+      - name: Install dependencies
+        run: pnpm install --filter "@quary/root"
```

Comment:
This only slows the job down because the make command uses npx

Reply:
We introduce `prettier-plugin-tailwindcss`, npx prettier depends on `prettier-plugin-tailwindcss` to work. The pnpm install step makes sure that this dependency is installed.